src/shared/DMIParser.c

```diff
@@ -0,0 +1,100 @@
+/*
+ * Created by Aavild on 19/11/2021.
+ */
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+#include <cJSON.h>
+#include "DMIParser.h"
+/*#include <curl/curl.h>*/
+
+/*input ninjo2dnudk.json and you will get out an array of windValues and a timestamp for last update*/
+windValue* ParseStringToWind(const char *input, int *lastUpdate)
+{
+    cJSON *json = cJSON_Parse(input);
+    if (json == NULL)
+    {
+        const char *error = cJSON_GetErrorPtr();
+        if (error != NULL)
+        {
+            printf("Error at: %s", error);
+        }
+        return 0;
+    }
+
+    cJSON *lastUpdateJson = cJSON_GetObjectItem(json, "lastupdate");
+    if (cJSON_IsString(lastUpdateJson) && (lastUpdateJson->valueint != 0))
+    {
+        printf("Input file last updated: \"%i\"\n", lastUpdateJson->valueint);
+    }
+    /*Test hertil i første omgang*/
+    windValue *values = malloc(sizeof(windValue)*95); /*timeserie of size 95*/
+    if(values == NULL)
+        return 0;  /* out of memory! */
```

Comment:
assert here and do the malloc+assert in one line (can wait till tests are being written)